cmd/maya-exporter/app/command/exporter.go

```diff
@@ -22,15 +19,20 @@ import (
 // from the requested volume. You need to pass the controller IP using flag -c
 // at runtime as a command line argument. Type maya-exporter -h for more
 // info.
-func Entrypoint(options *VolumeExporterOptions) {
-	controllerURL, err := url.Parse(options.ControllerAddress)
-	if err != nil {
-		log.Fatal(err)
+func Initialize(options *VolumeExporterOptions) string {
+	switch option := options.StorageEngine; option {
+	case "jiva":
+		return "jiva"
+	case "cstor":
+		return "cstor"
+	default:
+		return ""
```

Comment:
we are returning string and comparing this at other side. 
```
if option = "" {
  // exit and print "maya-exporter only supports jiva and cstor as storage engine"
}
if option = "jiva" {
 // register the exporter 
}
if option = "cstor" {
 // register exporter
}
// start http server
```
i have done this because for cstor we will be reading from the sockets but for jiva we will use rest apis. So initialize function returns the flag values passed along with the binaries. If we get values other than "jiva" and "cstor". It will return "".